source/elements/oneDNN/include/dnnl.hpp

```diff
@@ -1035,15 +966,6 @@ struct memory {
     ///       buffer.
     memory(const desc &md, const engine &aengine, void *handle);
```

Comment:
we should document that underlying buffer type is device usm

in the product we will state something like this: the underlying buffer depends on the runtime (for ocl -- cl_mem, for sycl -- device usm, etc)

Reply:
Included clarifications for `handle` and for the second constructor, that the underlying buffer will be `usm_device` if the user does not provide any memory.